utils/goDotEnv.go

```diff
@@ -0,0 +1,16 @@
+package utils
+
+import (
+	"log"
+	"github.com/joho/godotenv"
+)
+
+func setUpEnv() {
+
+	err := godotenv.Load("../.env")
+
+	if err != nil {
+		log.Panic("Error loading .env file")
```

Comment:
NIT: 
panic will also stop the flow of the command.
But since it is only being used in dev mode, it shouldn't cause an issue.

Any line that was written after panic won't work and the execution will stop

![image](https://user-images.githubusercontent.com/57758447/235089506-f2fbe9ce-2e6f-478e-8185-5247308fdb34.png)

Reply:
Got it.

Reply:
Sure, will make the changes with the next PR.